src/utils/products.expiration.ts

```diff
@@ -1,19 +1,18 @@
-import { sendEmail } from "../helpers/nodemailer";
 import { findAllProducts } from "../services/products.services";
 import { ProductAttributes, UserModelAttributes } from "../types/model";
 import { insert_function, read_function } from "./db_methods";
-import { update_product_email_template } from "./html.utils";
 import { EventName, myEmitter } from "./nodeEvents";
 
 export const checkProductExpiration = async () => {
 	const products: ProductAttributes[] = await findAllProducts();
 	const expiredProducts = products.filter((product) => {
 		const expirationDate = new Date(product.expiryDate);
 		const currentDate = new Date();
+
 		return (
 			expirationDate <= currentDate &&
 			product.quantity > 0 &&
-			product.productStatus === "Available"
+			product.isAvailable === true
```

Comment:
remove this "product.isAvailable === true" and have only 
"product.quantity > 0 &&
			product.isAvailable"